src/q_and_a_service/controller.js

```diff
@@ -1,14 +1,39 @@
 const pool = require ('../../db.js');
 const queries = require ('./queries.js');
 
-
 const getQA = (req, res) => {
-  pool.query(queries.getQA , (error, results) => {
+  console.log('queries.qa', queries.getQA);
```

Comment:
Is this `console.log` still needed? If not, you might consider removing it

Reply:
You are right.  I will delete it.